conan/tools/sbom/cyclonedx.py

```diff
@@ -99,3 +99,103 @@ def _calculate_licenses(component):
         "version": 1,
     }
     return sbom_cyclonedx_1_4
+
+def cyclonedx_1_6(conanfile, name=None, add_build=False, add_tests=False, **kwargs):
+    """
+    (Experimental) Generate cyclone 1.6 SBOM with JSON format
+
+    Creates a CycloneDX 1.6 Software Bill of Materials (SBOM) from a given dependency graph.
+
+
+
+    Parameters:
+        conanfile: The conanfile instance.
+        name (str, optional): Custom name for the metadata field.
+        add_build (bool, optional, default=False): Include build dependencies.
+        add_tests (bool, optional, default=False): Include test dependencies.
+
+    Returns:
+        The generated CycloneDX 1.6 document as a string.
+
+    Example usage:
+    ```
+    cyclonedx(conanfile, name="custom_name", add_build=True, add_test=True, **kwargs)
```

Comment:
This should also be updated for the 1_4, I think
```suggestion
    cyclonedx_1_6(conanfile, name="custom_name", add_build=True, add_test=True, **kwargs)
```